canonical-web-and-design/ubuntu.com
# Add the image-template on homepage

## Done
Added the image-template to the project and used it for all the images on the homepage.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Open your network tab in the inspector
- See that images file size has dropped from **370KB** to **60KB**
